### DISCUSSION TITLE
How to add comments?

### DISCUSSION THREAD

**Discussion by adi-itgg** [charleskorn/kaml#280] [Q&A] [answered]
How to add comments via class and make yaml with comments?

Expectations like this

```
@Serializable
data class Config(
    @Comment("version the config")
    val version: Int = 1,
    @Comment("update notification")
    val updateNotification: Boolean = true
)
```

**Comment by charleskorn** [comment]
Serializing comments currently isn't supported. If you're keen to see this feature, PRs are always welcome :)

**Reply by adi-itgg** [reply]
Okay thanks for the reply.

I'm just a little interested

nice library btw

**Comment by charleskorn** [comment, marked as answer]
Support for comments was added by @slava110 in #287, and released as part of [v0.45.0](https://github.com/charleskorn/kaml/releases/tag/0.45.0).
